java/src/com/swiftnav/sbp/drivers/SBPDriverHTTP.java

```diff
@@ -0,0 +1,44 @@
+package com.swiftnav.sbp.drivers;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+
+public class SBPDriverHTTP extends SBPDriverStream {
+    private final URL url;
+    private final String uid;
+
+    public SBPDriverHTTP(String uid_, String url_) throws MalformedURLException {
+        url = new URL(url_);
+        uid = uid_;
+    }
+
+    @Override
+    protected void openStream() throws IOException {
+
+        System.out.println("Connecting to " + url.toString());
+
+        HttpURLConnection http = (HttpURLConnection)url.openConnection();
```

Comment:
As it is now, if the connection breaks, the iterator will stop iteration and the forwarder thread will exit.  I guess we should probably handle timeouts and retries here, and require the application to interrupt the forwarder thread if it's no longer needed.